tests/omni_lock_rust/tests/misc.rs

```diff
@@ -81,6 +83,18 @@ pub const ERROR_SUPPLY_AMOUNT: i8 = 91;
 pub const ERROR_BURN: i8 = 92;
 pub const ERROR_NO_INFO_CELL: i8 = 93;
 
+// https://github.com/bitcoin-core/secp256k1/blob/d373bf6d08c82ac5496bf8103698c9f54d8d99d2/include/secp256k1.h#L219
+pub const SECP256K1_TAG_PUBKEY_EVEN: u8 = 0x02;
+pub const SECP256K1_TAG_PUBKEY_ODD: u8 = 0x03;
+pub const SECP256K1_TAG_PUBKEY_UNCOMPRESSED: u8 = 0x04;
+pub const SECP256K1_TAG_PUBKEY_HYBRID_EVEN: u8 = 0x06;
+pub const SECP256K1_TAG_PUBKEY_HYBRID_ODD: u8 = 0x07;
+
+pub const BITCOIN_V_TYPE_P2PKHUNCOMPRESSED: u8 = 27;
+pub const BITCOIN_V_TYPE_P2PKHCOMPRESSED: u8 = 31;
+pub const BITCOIN_V_TYPE_SEGWITP2SH: u8 = 35;
+pub const BITCOIN_V_TYPE_SEGWITBECH32: u8 = 39;
```

Comment:
The names look wierd and are not consistent with the constants above.

Reply:
note, they're from bitcoin, not secp256k1. I will add some comments.